app/models/content_item.rb

```diff
@@ -1,6 +1,7 @@
 class ContentItem < ApplicationRecord
   has_and_belongs_to_many :organisations
   has_and_belongs_to_many :taxons
+  has_and_belongs_to_many :terms
```

Comment:
Does the ContentItem have terms, or is it the TaxonomyTodo? Feels like the latter to me, and we could avoid change to core classes.